README.md

```diff
@@ -14,24 +14,18 @@ comprehensive set of [common tools](https://Redotengine.org/features), so that
 users can focus on making games without having to reinvent the wheel. Games can
 be exported with one click to a number of platforms, including the major desktop
 platforms (Linux, macOS, Windows), mobile platforms (Android, iOS), as well as
-Web-based platforms and [consoles](https://docs.Redotengine.org/en/latest/tutorials/platform/consoles.html).
+Web-based platforms and [consoles](https://Redotengine.org).
 
 ## Free, open source and community-driven
 
 Redot is completely free and open source under the very permissive [MIT license](https://Redotengine.org/license).
 No strings attached, no royalties, nothing. The users' games are theirs, down
 to the last line of engine code. Redot's development is fully independent and
 community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the [Redot Foundation](https://Redot.foundation/)
```

Comment:
The Redot Foundation doesn't exit yet... so we should probably cut this line for now.